utils/statserial/Makefile

```
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=statserial
+PKG_VERSION:=1.1-25
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://salsa.debian.org/debian/statserial.git
+PKG_SOURCE_VERSION:=debian/$(PKG_VERSION)
+PKG_MIRROR_HASH:=d493fd3154de4e52811e1d4f2ee68d573b1bcc24b58089b3b3dc41461357920e
+
+PKG_MAINTAINER:=Jakub Raczynski <myszsoda@gmail.com>
+PKG_LICENSE:=GPL-2.0-only
```

```suggestion
PKG_LICENSE:=GPL-2.0-or-later
```

See [statserial.c](https://salsa.debian.org/debian/statserial/-/blob/debian/1.1-25/statserial.c#L8-9).